Makefile.js

```diff
@@ -365,6 +392,7 @@ target.test = function() {
     }
 
     target.checkLicenses();
+    target.checkGitCommit();
```

Comment:
I think what he proposed was to remove this line from here as this would be called whenever we run `npm test`.
Instead expose `checkGitCommit` through package script tag and then call this inside post test steps in travis.